packages/smooth_app/lib/pages/image_crop_page.dart

```diff
@@ -32,19 +32,22 @@ Future<File?> startImageCropping(BuildContext context) async {
       CropAspectRatioPreset.ratio4x3,
       CropAspectRatioPreset.ratio16x9
     ],
-    androidUiSettings: AndroidUiSettings(
-      toolbarTitle: appLocalizations.product_edit_photo_title,
-      initAspectRatio: CropAspectRatioPreset.original,
-      lockAspectRatio: false,
-      statusBarColor: themeColor,
-      toolbarColor: themeColor,
-      toolbarWidgetColor: Colors.white,
-      activeControlsWidgetColor: Theme.of(context).colorScheme.primary,
-      backgroundColor: themeColor,
-    ),
-    iosUiSettings: const IOSUiSettings(
-      minimumAspectRatio: 1.0,
-    ),
+    // ignore: always_specify_types
```

Comment:
Please keep the `ignore` for very good reasons.
Here it doesn't look like as a good reason.